### ISSUE TITLE
How does one get started using flask-easy in their own projects?

### ISSUE THREAD

**Issue by mickeymond** [Josephmaclean/flask-easy#7]
1. Do you intend for others to use it in their own projects?
2. How do they get to do that?

**Comment by Josephmaclean** [comment]
Yes. This is an opensource project I intend for others to use in their own projects
Will publish the latest version on pypi later today. 

Also I'm yet to document it so I guess that's why you are asking this question. You can open a pull request to do just that.
